R/filters.R

```diff
@@ -94,9 +94,20 @@ read_long_indicator_filters <- function(data, type) {
 }
 
 get_model_output_filters <- function(data) {
-  list(age = get_age_filters(data),
-       quarter = get_quarter_filters(data),
-       indicators = get_id_label_map(data, "indicator_id"))
+  list(
+    list(
+      id = scalar("age"),
+      column_id = scalar("age_group_id"),
```

Comment:
@hillalex Could you review this section?

I'm interpreting the meaning of

id - a machine readable id for the filter does this need to match anything in particular?
column_id - the name of the column to filter on
label - human readable label for the filter
options - label, id pairs for filters where label is thing to be shown to the user and the id matches the id of one of the values within the column `column_id`.

Is that right?

Reply:
Yes that's exactly right. The filter id is something we use on our side to identify filters internally, does not need to match anything.